### DISCUSSION TITLE
How to get patches coordinates from RandCropByPosNegLabeld?

### DISCUSSION THREAD

**Discussion by OeslleLucena** [Project-MONAI/MONAI#1751] [Q&A]
Is there a way I can get the coordinates for all patches sampled from `RandCropByPosNegLabeld`?
After the forward pass, I can see that the batch dict has `image_meta_dict` and `label_meta_dict` but I cannot find the coordinates from the sampled patches. I tried  `batch['image_meta_dict']['slice_start'])` and  `batch['image_meta_dict']['slice_end']` but I get [0,0,0,0] from both.

**Comment by rijobro** [comment]
The centres of the segmentations are stored within the class, but aren't returned to the user: https://github.com/Project-MONAI/MONAI/blob/master/monai/transforms/croppad/array.py#L607. This seems like it would be an easy modification to make. Feel free to create an issue as a feature request and I'll have a look at it.
